test/certifi_tests.erl

```diff
@@ -2,17 +2,27 @@
 
 -include_lib("eunit/include/eunit.hrl").
 
+%% Update this when updating priv/cacerts.pem.
+-define(NUM_OF_CERTS, 147).
```

Comment:
Initially, I thought this number could be generated by parsing `priv/cacerts.pem`.
However, I realized this approach would weaken the test because parsing the file to verify it is properly parsed sounds somewhat circular.